torch/csrc/jit/pybind_utils.h

```diff
@@ -167,6 +167,27 @@ inline IValue argumentToIValue(
   }
 }
 
+inline IValue returnToIValue(
+    size_t pos,
+    const TypePtr& type,
+    py::handle object) {
+  try {
+    return toIValue(object, type);
```

Comment:
Couldn't we replace all uses of this function with this/a similar helper? Our error messages are pretty bad at the moment.